server/call/session.py

```diff
@@ -283,6 +285,20 @@ def on_left_meeting(self, _, error: str = None):
         if error:
             self._logger.error(
                 "Encountered error while leaving meeting: %s", error)
+
+        # There's a chance of a shutdown timer being ongoing at the time the bot
+        # is kicked or leaves for other reasons. Clean up the shutdown timer if
+        # that is the case.
+        if self._shutdown_timer:
```

Comment:
This could happen if the call state changes to `"left"`  and `on_left_meeting()` is invoked that way.